### PR TITLE
Reflavors a lot of Bloodsucker stuff to be more in line with it's source

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.

## Why It's Good For The Game

My love of VTM cannot stand things such as 'blah-sucker', 'vassels' or 'archive of the kindred'.

The Blah-Sucker language has been renamed to Enochian, reflecting the First City of Vampires, Enoch, in the land of Nod east of Eden where the Dark Father Caine was exiled from.

(Front facing) mentions of vassels have been replaced with the accurate Ghoul. They drink vitae to establish a bond with their master, not blood, vampires do not produce blood, they drink vitae.

The Archive of the Kindred has been replaced with the Book of Nod. Well, a version of it. Cainites are protective of the hidden knowledge kept within and would be unhappy with the mortals (you) having a copy.

Objectives have been re-written to mention lore aspects.

## Proof Of Testing

:cl:
spellcheck: Various Bloodsucker things have been renamed after the Inconnu paid a visit.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/objectives.dm [hunk 1]
```diff
@@ -247,7 +247,7 @@
 
 // EXPLANATION
 /datum/objective/bloodsucker/tremere_power/update_explanation_text()
-	explanation_text = "Upgrade a Blood Magic power to the maximum level, remember that Vassalizing gives more Ranks!"
+	explanation_text = "Your Regent is doubting your abilities, level some Blood Magic to prove them wrong! Remember that Vassalizing gives more Ranks!"
```
Comment (theselfish) [inline modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/objectives.dm:250]:
```suggestion
	explanation_text = "Your Regent is doubting your abilities, level some Blood Magic to prove them wrong! Remember that Ghoulifying gives more Ranks!"
```

**Comment by Arturlang** [comment]
If you're going to do this, do it properly, you're still missing a lot of mentions, I'm quite sure. Use VSCode's mass replace

**Comment by destrucktoid** [comment]
Based and VTM pilled. I am all for this

**Comment by theselfish** [comment]
I dunno how to run prettier.

**Comment by shadaondiscord** [comment]
Go all in on this and make it so you can use your vitae to recruit people by getting them to drink it. That is, if your vitae touches the stomach container before the bloodstream. This also makes it easier for a lot of people to RP as getting recruited willingly because you aren't signing up to be scarified under a welding torch you're just drinking something, a lot easier of an action for people to commit.

**Comment by Arturlang** [comment]
> I dunno how to run prettier.

You can install the VSCode prettier extension or run it via npm, npx prettier

**Comment by theselfish** [comment]
> > I dunno how to run prettier.
> 
> You can install the VSCode prettier extension or run it via npm, npx prettier in the commandline, that assumes you have npm installed

....Yeah no idea. Tried that, nothin.

**Comment by Erol509** [comment]
> > > I dunno how to run prettier.
> > 
> > 
> > You can install the VSCode prettier extension or run it via npm, npx prettier in the commandline, that assumes you have npm installed
> 
> ....Yeah no idea. Tried that, nothin. The VSC route, not got NPX.

Because TG uses a special unofficial version of prettier, other versions are not suported.

**Comment by theselfish** [comment]
> > > > I dunno how to run prettier.
> > > 
> > > 
> > > You can install the VSCode prettier extension or run it via npm, npx prettier in the commandline, that assumes you have npm installed
> > 
> > 
> > ....Yeah no idea. Tried that, nothin. The VSC route, not got NPX.
> 
> Because TG uses a special unofficial version of prettier, other versions are not suported.

Well I'm unsure how to fix the linter, then.

**Comment by theselfish** [comment]
💀

**Comment by theselfish** [comment]
Updates arghaaaa!!!!

**Pull request closed without merging**